AirCasting/BluetoothSession/BluetoothSessionRecordingController.swift

```diff
@@ -56,33 +57,39 @@ class MobileAirBeamSessionRecordingController: BluetoothSessionRecordingControll
         Resolver.resolve(AirBeamConfigurator.self, args: device)
             .configureMobileSession(location: locationTracker.location.value?.coordinate ?? .undefined,
                                     completion: completion)
+        
         // Info: we're not changing the sessions `status` property here to `.RECORDING` because it is currently
         // being done by the MeasurementStreamStorage class automagically.
         // This is something we might want to change at some point.
```

Comment:
is this actionable?

Reply:
I think this can possibly be done during MeasurementStramStorage refator